web-client/public/js/graph.js

```diff
@@ -49,10 +48,11 @@ var drawGraph = function (nodes, links, positiveWeights, negativeWeights, sheetT
 
     var MIN_SCALE = 0.25;
     var ADAPTIVE_MAX_SCALE = 4;
+    var MIDDLE_SCALE = 1;
     // regardless of whether the viewport is fixed or adaptive, the zoom slider now operates on the same scale
 
     var minimumScale = MIN_SCALE;
-    var maximumScale = ADAPTIVE_MAX_SCALE;
+    // var maximumScale = ADAPTIVE_MAX_SCALE;
```

Comment:
If you're sure you don't need this anymore, then delete the line outright; don't leave it commented out.

Reply:
Deleted.